.github/workflows/cd-create-release-pr.yaml

```diff
@@ -0,0 +1,90 @@
+name: Create Release PR
+on:
+  workflow_dispatch:
+    inputs:
+      bump:
+        type: choice
+        description: The version bump type.
+        default: minor
+        options:
+          - major
+          - minor
+          - patch
+
+jobs:
+  create-release-pr:
+    runs-on: ubuntu-latest
+    environment:
+      name: "RELEASE_CREATION"
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: ./.github/actions/setup-python-env
+
+      - name: Use Changie to Generate Change Log for the New Release
+        uses: miniscruff/changie-action@v2
+        with:
+          version: latest
+          args: batch ${{ inputs.bump }}
+
+      - name: Use Changie to Update `CHANGELOG.md` To Include All Changes
+        uses: miniscruff/changie-action@v2
+        with:
+          version: latest
+          args: merge
+
+      - name: Use Changie to Get the New Version Number
+        id: changie-latest
+        uses: miniscruff/changie-action@v2
+        with:
+          version: latest
+          args: latest
+
+      - name: Save the New Version Into Context
+        id: get-new-package-version
+        run: |
+          VERSION_WITH_V_PREFIX="${{ steps.changie-latest.outputs.output }}"
+          VERSION=$(echo "VERSION_WITH_V_PREFIX" | cut -c 2-)
+          echo "version=$VERSION" >> $GITHUB_OUTPUT
+
+      - name: Update Project Version in Source Code via Hatch
+        id: update-package-version
+        run: |
+          hatch version "${{ steps.get-new-package-version.outputs.version }}"
```

Comment:
Should we add a sanity check here that ensures `"$(hatch version)" = "${{ steps.changie-latest.outputs.output }}"`?

This _shouldn't_ ever happen, but in case something somehow gets borked, we make sure we never open a PR with mismatched versions in `CHANGELOG.md`, `pyproject.toml` and `__about__.py`.